### DISCUSSION TITLE
How do I get the menu line of the text editor back?

### DISCUSSION THREAD

**Discussion by pnachtwey** [linuxmint/discussions#92] [Questions]
The menu line is missing.   I need to enable seeing hidden files so I can edit the .profile  and .bachrc file.
All I see is icons at the top of the text editor.   There is no icon that enables viewing hidden files.

**Comment by Linuxmonger** [comment]
For me, in Linux Mint 21.2 Cinnamon, tapping the Alt key toggles the Menu line in Text Editor.
After that line is visible, click on View, and check the Option Box for Menu Bar.
   or
Entirely from the keyboard, hold down Alt, when the first letter of each Menu item is underlined, tap 'V', release Alt and tap 'M', followed by the Space to toggle the Option Box for Menu Bar.

I'm not sure if this resolves the issue, or if you have two issues, If you have an issue viewing hidden files as well, in the file browser Nemo, tap Ctrl-H on the keyboard to toggle the ability to view hidden files.
